packages/easysearch:core/lib/core/search-collection.js

```diff
@@ -184,20 +186,47 @@ class SearchCollection {
       this.added(collectionName, 'searchCount' + definitionString, { count });
 
       let intervalID;
-
       if (collectionScope._indexConfiguration.countUpdateIntervalMs) {
+        intervalID = Meteor.setInterval(() => {
+            let newCount;
```

Comment:
@matteodem It seems that the `cursor.count()` returns the initial count value set while instantiating the cursor, so I added a optional `mongoCount` parameter that is set to `false` by the `ESCursor`. Is this a satisfied solution or should I do it otherwise?